docs/openapi/components/schemas/credentials/BankAccountCredential.yml

```diff
@@ -77,12 +82,13 @@ required:
   - type
   - issuanceDate
   - issuer
-  - credentialSubject    
+  - credentialSubject
 example: |-
   {
     "@context": [
       "https://www.w3.org/2018/credentials/v1",
-      "https://w3id.org/traceability/v1"
+      "https://w3id.org/traceability/v1",
+      "https://w3id.org/vc/status-list/2021/v1"
     ],
     "id": "did:key:1111111",
```

Comment:
```suggestion
    "id": "urn:uuid:123....",
```

Reply:
^ probably want to do this everywhere

Reply:
we originally had `did:knox` but since that wasn't on the did method registry yet it was changed to `key`.  BTW- the did:knox submission is here now 🙂
https://github.com/w3c/did-test-suite/pull/223